docs/attributes-registry/db.md

```diff
@@ -65,62 +73,62 @@ Semantic conventions for individual database systems SHOULD document what `db.re
-| `sqlite`             | SQLite                                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
-| `sybase`             | Sybase                                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
-| `teradata`           | Teradata                                                  | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
-| `trino`              | Trino                                                     | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
-| `vertica`            | Vertica                                                   | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| Value                | Description                                                             | Stability                                                                                        |
+| -------------------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
+| `adabas`             | Adabas (Adaptable Database System)                                      | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `cache`              | Deprecated, use `intersystems_cache` instead.                           | ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>Replaced by `intersystems_cache`. |
+| `cassandra`          | Apache Cassandra                                                        | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `clickhouse`         | ClickHouse                                                              | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `cloudscape`         | Deprecated, use `other_sql` instead.                                    | ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>Replaced by `other_sql`.          |
+| `cockroachdb`        | CockroachDB                                                             | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
+| `coldfusion`         | Deprecated, no replacement at this time.                                | ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>Removed.                          |
+| `cosmosdb`           | Microsoft Azure Cosmos DB                                               | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
```

Comment:
```suggestion
| `cosmosdb`           |  Azure Cosmos DB (This value has stability level RELEASE CANDIDATE)                                              | ![Experimental](https://img.shields.io/badge/-experimental-blue)                                 |
```

Reply:
Can we also make `cosmosdb` a release candidate in the database-spans and database-metrics and registry files? It won't let me leave a comments on those lines since they weren't edited in this PR.

Reply:
@lmolkova @jcocchi and I discussed this offline and we're going to pursue stabilizing cosmosdb, but not include it in this initial release candidate